common/src/stack/report-system/command/report/system/tests/test_other_dhcp.py

```diff
@@ -57,7 +57,27 @@ def test_other_dhcp_server(interface):
 	# Set up socket with passed in interface
 	send_dhcp = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
 	send_dhcp.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
-	send_dhcp.setsockopt(socket.SOL_SOCKET, 25, str(interface + '\0').encode('utf-8'))
+
+	# Bail if we can't setup a socket on the current interface
```

Comment:
I believe you know at least one case where this is true, might as well mention it here.